libs/utils/utils.js

```diff
@@ -91,12 +94,12 @@ function getEnv(conf) {
 // find out current locale based on pathname and existing locales object from config.
 export function getLocale(locales) {
   if (!locales) {
-    return { ietf: 'en-US', tk: 'hah7vzn.css', prefix: '' };
+    return { ietf: 'en-US', tk: 'hah7vzn.css', prefix: 'en' };
   }
   const { pathname } = window.location;
   const split = pathname.split('/');
   const locale = locales[split[1]] || locales[''];
-  locale.prefix = locale.ietf === 'en-US' ? '' : `/${split[1]}`;
+  locale.prefix = locale.ietf === 'en-US' ? 'en' : `/${split[1]}`;
```

Comment:
This needs to be changed back. There's reasons we have it a certain way. I can work with you on the blog use case.